test/conftest.py

```diff
@@ -10,8 +12,15 @@
 
 @pytest.fixture(autouse=True, scope='function')
 def pre_and_post_test():
+    # Set a dummy google cloud project to avoid errors when running tests for tests
+    # that use the google cloud.
+    os.environ["GOOGLE_CLOUD_PROJECT"] = "dummy-project-for-tests"
```

Comment:
you could set/unset this using a monkeypatch

Reply:
I replaced this with a `mock.patch.dict`. It's just a temporary solution while we figure out how to properly test production pipelines.